Coursework/2-Functions/exercise5-question.js

```diff
@@ -8,6 +8,11 @@ We should write a function, and call the function 3 times.
 The results should stay THE SAME
 */
 
+function greeting(){
+    let greeting = firstName1;
+    return greeting;
+}
+
```

Comment:
For this you'll need to create a function that replicates the basic logic of lines 19, 24 and 29 in order to build a string.

You should end up with a function that looks something like this:

```
function greeting() {firstName, surname} {
    return ...;
}
```

`...` above will be the code that builds your string by concatenating the various parts. Then you'll be able to do say `let greeting1 = greeting(firstName1, surname1)`.